### PR TITLE
#184581322 Circleci project setup

### PR DESCRIPTION
**Pull request by bencyubahiro77**
#### What does this PR do?
Integrate CircleCi with readme badge

#### Description of Task to be completed?

- Created CircleCi account with the Github account
- Joined the repo branch with CircleCi account to with **build** and **test** job.
- Added CircleCi URL in the Readme file to have a badge above it.

#### How should this be manually tested?

- it can be tested by clicking on the badge above the readme file which direct  you to the CircleCi account for more details.

#### Any background context you want to provide?
 For the success integration of CircleCi they will be green badge and red for the failed integration which will present the error caused it to fail.

#### What are the relevant pivotal tracker/Trello stories?
 #184581322

#### Questions:

### CODE REVIEW

**Review by blackd44 — COMMENTED** [review]
Benjamin try to resolve the conflict,
others are fine to me

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  sample: # This is the name of the workflow, feel free to change it to better match your workflow.
+    # Inside the workflow, you define the jobs you want to run.
+    jobs:
+      - node/test:
+          # This is the node version to use for the `cimg/node` tag
+          # Relevant tags can be found on the CircleCI Developer Hub
+          # https://circleci.com/developer/images/image/cimg/node
+          version: '16.10'
+          # If you are using yarn, change the line below from "npm" to "yarn"
+          pkg-manager: npm
```
Comment (Nkbtemmy) [inline .circleci/config.yml]: As your task was to do circlesCi integration; you didn't do any job required, you need to do jobs like build and test and  also remember to add workflow in this file, and remember to reduce those comments they make this configuration not look good

**Review by Nkbtemmy — CHANGES_REQUESTED** [review]

**Review by Nkbtemmy — APPROVED** [review]

File: .circleci/config.yml (hunk 1 above)
Comment (bencyubahiro77) [reply .circleci/config.yml]: added jobs . you can check now if it is ok!!

**Review by Martha-Iradukunda — APPROVED** [review]
LGTM

**Comment by edwinnambaje** [comment]
LGTM

**Review by edwinnambaje — APPROVED** [review]
LGTM

**Review by nshutijonathan — CHANGES_REQUESTED** [review]
fix the conflicts ,README.md

**Review by edwinnambaje — APPROVED** [review]
LGTM

**Review by nshutijonathan — CHANGES_REQUESTED** [review]
Please make  sure to squash your commits  into  one

**Review by nshutijonathan — APPROVED** [review]
LGTM

**Pull request merged**